helm/templates/logger-service.yaml

```diff
@@ -7,6 +7,8 @@ spec:
   ports:
   - name: "8500"
     port: 8500
+  - name: "6379"
+    port: 6379
```

Comment:
the redis port should be exposed only if redis is hosted on the logger (there is a value controlling this)